lua/lualine/components/mode.lua

```diff
@@ -2,20 +2,34 @@ local function mode()
   local mode_map = {
     ['__'] = '------',
```

Comment:
funny... I don't recall what this is supposed to do... nothing I guess? pretty sure it's safe to delete

Reply:
I could't understand why it was there either.